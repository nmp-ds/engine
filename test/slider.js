import { setup } from './_helpers.js';
import { describe, expect, test } from 'vitest';

setup();

describe('slider', () => {
  test('get active slider shadow', async ({ uno }) => {
    const { css } = await uno.generate(['slider-handle-shadow-active']);
    expect(css).toMatchSnapshot();
  });

  test('get hover slider shadow', async ({ uno }) => {
    const { css } = await uno.generate(['slider-handle-shadow-hover']);
    expect(css).toMatchSnapshot();
  });

  test('get slider outline none', async ({ uno }) => {
    const { css } = await uno.generate(['slider-handle-outline-none']);
    expect(css).toMatchSnapshot();
  });
});
