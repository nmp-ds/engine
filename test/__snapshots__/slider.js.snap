// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`slider > get active slider shadow 1`] = `
"/* layer: default */
.slider-handle-shadow-active{box-shadow:0 0 0 8px rgba(0, 0, 0, .08);}"
`;

exports[`slider > get hover slider shadow 1`] = `
"/* layer: default */
.slider-handle-shadow-hover{box-shadow:0 0 0 6px rgba(0, 0, 0, .08);}"
`;

exports[`slider > get slider outline none 1`] = `
"/* layer: default */
.slider-handle-outline-none{outline:rgba(0, 0, 0, 0) solid 2px;outline-offset:2px;}"
`;
